apps/frontend/src/pages/LandingPage.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Box, Button, Image, VStack } from '@chakra-ui/react';
+
+function AuthButtons() {
+  /* Temporary styles for buttons; to be replaced with Chakra 'useStyleConfig'. */
+  const buttonStyles = {
+    height: '4em',
+    width: { base: '50%', md: '50%' },
+    fontSize: { base: 'lg', md: 'xl' },
+  };
+
+  return (
+    <VStack w={'100%'} spacing={4}>
+      <Link to="/signup">
+        <Button
+          h={buttonStyles.height}
+          w={buttonStyles.width}
+          fontSize={buttonStyles.fontSize}>
+            SIGN UP
+        </Button>
+      </Link>
+      <Link to="/register">
+        <Button
+          h={buttonStyles.height}
+          w={buttonStyles.width}
+          fontSize={buttonStyles.fontSize}>
+            LOGIN
+        </Button>
+      </Link>
```

Comment:
Did you want to merge this commit, then I can update my branch to remove potential conflicts, then I can move the `LoginPage` from `components` to `pages` and update the routing?